pkg/apis/componentconfig/types.go

```diff
@@ -503,6 +503,8 @@ type KubeControllerManagerConfiguration struct {
 	// horizontalPodAutoscalerSyncPeriod is the period for syncing the number of
 	// pods in horizontal pod autoscaler.
 	HorizontalPodAutoscalerSyncPeriod unversioned.Duration `json:"horizontalPodAutoscalerSyncPeriod"`
+	// URL to an external heapster backend
+	Heapster string `json:"heapster"`
```

Comment:
What's wrong with HeapsterUrl?

If the concern is that it's too specific (i.e.this will actually work with any monitoring aggregator that exports the metrics API) then maybe you could call it `MetricsSource` or something like that?